common/app/com/commercetools/sunrise/play/SunriseHttpFilters.java

```diff
@@ -1,5 +1,7 @@
-package com.commercetools.sunrise.common.basicauth;
+package com.commercetools.sunrise.play;
```

Comment:
You are wrong, I also use the strategy you mention. But in this case it's different.

What I want to achieve with this is not having so many packages in `common` with just one class inside, especially since all these classes are unique and won't have more `Filters`, more `Configuration` or more `ApplicationLoader`. They are just replacing the default stuff in Play. If you tell me there are at least two filters/configuration/application-loaders related classes, then I'll agree with you.

Of course then we have `BasicAuth` which is using the filter strategy and could go inside the `filters` package, but this is just the current implementation, just a few weeks ago it was using another strategy. That's why I would like to keep this outside any folder referencing `filters`.